core/src/main/java/UIElements/TriviaButtonBuilder.java

```diff
@@ -25,34 +24,50 @@
 
```

Comment:
I needed the render and create methods. I'll check if there is a class that has these methods only.